src/cocoa/toga_cocoa/widgets/base.py

```diff
@@ -14,6 +14,18 @@ def _set_container(self, container):
             self._constraints._container = container
         self.rehint()
 
+    def _set_hidden(self, status, children):
+        for view in self._container._impl.subviews:
```

Comment:
Iterating over the list of children should be something common to *all* containers; could this be moved up to the interface layer?